pixeltable/exprs/expr.py

```diff
@@ -495,6 +495,18 @@ def eval(self, data_row: DataRow, row_builder: 'exprs.RowBuilder') -> None:
         """
         pass
 
+    def prepare(self) -> None:
+        """
+        Create execution state. This is called before the first eval() call.
+        """
+        for c in self.components:
+            c.prepare()
+
+    @classmethod
+    def prepare_list(cls, expr_list: list[Expr]) -> None:
```

Comment:
This seems a little silly to me to have as a separate method, but if we're going to have it, it should take an Iterable.

Reply:
Done.